stats/Record.md

```diff
@@ -74,6 +77,12 @@ measurementMap.put(RPC_LATENCY, 10.3);
 measurementMap.put(RPC_BYTES_SENT, 124);
 measurementMap.record();  // reads context from thread-local.
 
+MeasurementMap measurementMap = new MeasurementMap();
+measurementMap.put(RPC_LATENCY, 15);
+measurementMap.put(RPC_BYTES_SENT, 200);
+measurementMap.record(
+    Tags.getTagger().currentBuilder().put(myKey, myValue).build());  // record against an extra tag.
```

Comment:
It might be clearer to split the code into four blocks, shared measure constants and three examples of recording stats.  Then some of the code comments could be moved above the blocks of code that they describe.

Reply:
Sounds good, updated.

Reply:
I meant that the code comment at the top could be moved outside of the code block, as a description for the whole code block, though the current formatting is also fine.